GoogleForCreators/web-stories-wp
# Implement plugin activation message

## Summary

Implements the plugin activation message according to the [designs](https://www.figma.com/file/HEVtT9QucEpRGvzUkCCvL6/Beta-Micro-Site?node-id=101%3A9) shared at #2853

## Relevant Technical Choices

* **For testing & review purposes only:** This PR currently includes a small theme switcher. **Double click on the message** to switch between light and dark theme.  
  This way we can get a better sense for which works best. Also, theoretically we can keep both themes and do A/B testing in the future.
  **This switcher will be removed before merge.**
* The original design does not work on a 15 inch screen. I took some liberty to make some assets **smaller**, and to add some **responsive design** to the whole thing.
* Ensures that all clicks to links within the message can be tracked.

## To-do

* [ ] Implement Get Started story (#2845)
* [ ] Remove theme switcher once settled for a theme.
* [ ] Visual fine tuning

## User-facing changes

Implements a new plugin activation message, check screenshots below:

<details>
<summary>Small Screen (Smartphone)</summary>

<img width="370" alt="Screenshot 2020-07-15 at 11 14 33" src="https://user-images.githubusercontent.com/841956/87542320-11462500-c6a3-11ea-880c-baf0d70fe147.png">
<img width="371" alt="Screenshot 2020-07-15 at 11 14 25" src="https://user-images.githubusercontent.com/841956/87542331-13a87f00-c6a3-11ea-82c9-d578d88b2e6b.png">

</details>

<details>

<summary>Small Tablet</summary>

<img width="784" alt="Screenshot 2020-07-15 at 11 15 04" src="https://user-images.githubusercontent.com/841956/87542382-258a2200-c6a3-11ea-8ffc-f368e5a7d2df.png">
<img width="783" alt="Screenshot 2020-07-15 at 11 15 14" src="https://user-images.githubusercontent.com/841956/87542372-228f3180-c6a3-11ea-8630-96a6893cfb7e.png">

</details>

<details>
<summary>Large Tablet</summary>

<img width="1178" alt="Screenshot 2020-07-15 at 11 15 42" src="https://user-images.githubusercontent.com/841956/87542422-33d83e00-c6a3-11ea-9ffb-9bc9a0b28799.png">
<img width="1178" alt="Screenshot 2020-07-15 at 11 15 34" src="https://user-images.githubusercontent.com/841956/87542432-376bc500-c6a3-11ea-8a0e-2d43764fc4fa.png">

</details>

<details>
<summary>Desktop</summary>

<img width="1680" alt="Screenshot 2020-07-15 at 11 15 56" src="https://user-images.githubusercontent.com/841956/87542455-405c9680-c6a3-11ea-989b-eb80f3413220.png">
<img width="1680" alt="Screenshot 2020-07-15 at 11 16 05" src="https://user-images.githubusercontent.com/841956/87542446-3cc90f80-c6a3-11ea-804f-83c4d6c12cee.png">

</details>

## Testing Instructions

1. Deactivate plugin.
1. Activate plugin.
1. Verify that all links work (except for Get Started story!)
1. Verify that images are clickable too
1. Verify that plugin activation message is responsive
1. On smaller screens, a "See sample story" link is displayed below the "Launch the editor"  button

---



Fixes #2853
